src/ert/config/gen_kw_config.py

```diff
@@ -58,18 +59,44 @@ def _get_abs_path(file: Optional[str]) -> Optional[str]:
     return file
 
 
+@dataclass
+class TransformFunctionDefinition:
+    func_name: str
```

Comment:
Can we drop the `func` from `fun_name` and `param_func_name`?

Reply:
I will drop the funk